Banners, Langauge, Commands, Times and Lists, oh my!

skylordjason opened the pull request:
**Not quite the same ring, but it's what was done.**

Added the following banner to the top of the Readme, because banners are awesome. (Or to make it look better)
![image](https://cloud.githubusercontent.com/assets/8324596/12280086/0c8d61f4-b951-11e5-899d-1d5452db1d17.png)

Added the ability for a user to do `@bot list all` to get a generated PM of all videos currently in the queue. Splits the list into multiple messages to fit Discord's Char Limit.

Modified the time that is shown on _Remaining Play Time:_ to say hours and minutes versus an unmodified time. (1 hour and 30 minutes over 90:00 minutes)

Modified the Language of some of the Readme to be more friendly or logical, whichever came first. 

Modified the Readme to represent the added command above as well as link to the Usage wiki for detailed usage info.

Modified the Readme to explain how the bot will initialize depending on your connected channel.

skylordjason committed:
Update README.md

skylordjason committed:
Update README.md

skylordjason committed:
Update README.md

skylordjason committed:
Update README.md

skylordjason committed:
Add a logo to ReadMe. Change a few things.

Just a logo idea for Lethe. Plus, a bit of changes to keep it up to date with current commands and features.

skylordjason committed:
Merge remote-tracking branch 'meew0/master'

skylordjason committed:
list all is a thing

This one is a necessity to PM, as it would flood the chat for large
queues.

skylordjason committed:
list all and invisible characters

meew0 commented on README.md:
````diff
@@ -23,18 +23,30 @@ npm install
 ```
 
 ## Usage
+####Run by Command Line Arguments
````
Could you follow the correct markdown hierarchy here? (Sub headers of a second-level header are always third-level)

meew0 commented on README.md:
````diff
@@ -23,18 +23,30 @@ npm install
 ```
 
-Then, run commands over Discord using the bot's username mention as a prefix, for example:
+####Run by pre-set credentials in lethe-auth.json
````
Same here

meew0 commented on lethe.js:
```diff
@@ -349,7 +396,18 @@ client.on('message', m => {
         }
       });
-      formattedList += `\n**Remaining play time:** ${Util.formatTime(overallTime)} minutes.`;
+	  var hoursOf = Math.floor(overallTime / 60 / 60);
```
Could you make this code into a separate function so 1) the commands are less cluttered and 2) there's no duplicate code (same function should be used in `list all`)?

meew0 replied:
Or just modify `formatTime` directly

meew0 commented:
Also, the problem with the `list all` command is that even though it's sent over PM, Discord's rate limiting still applies so for _very_ long queues it will eventually break. I suggest that if it gets too long (>3 messages) it gets sent to hastebin instead like `list saved` does.

skylordjason committed:
Formatting, because whoops

When you intended to have something else there for the 3rd heading, but then decided against it and never reverted it.

skylordjason committed:
Testing haste (don't accept)
